autoload/jsdoc.vim

```diff
@@ -111,7 +111,7 @@ function! jsdoc#insert()
 
   let l:line = getline('.')
   let l:indentCharSpace = ' '
-  let l:indentCharTab = '	'
+  let l:indentCharTab = '       '
```

Comment:
You should not replace tab to space.
`:set noexpandtab` settings user used `l:indentCharTab`.